src/datepicker/utils/date-helpers.js

```diff
@@ -40,7 +40,7 @@ class DateHelpers<T> {
           monthNumber: 'M',
           dayOfMonthNumber: 'd',
           weekday: 'EEEE',
-          weekdaymin: 'EEEEE',
+          weekdaymin: 'EEEEEE',
```

Comment:
Here's I'm changing the weekday headers from "S M T W..." to "Su Mo Tu We...".  I'm not 100% sure if this is the only place I should be making this change.